src/components/organisms/AdminVenueView/AdminVenueView.tsx

```diff
@@ -0,0 +1,108 @@
+import React, { useCallback, useState } from "react";
+import { Button, Nav } from "react-bootstrap";
+import { Link } from "react-router-dom";
+import classNames from "classnames";
+
+import AdvancedSettings from "pages/Admin/AdvancedSettings";
+import BasicInfo from "pages/Admin/BasicInfo";
+import EntranceExperience from "pages/Admin/EntranceExperience";
+import VenueDetails from "pages/Admin/Venue/Details";
+
+import { Venue_v2 } from "types/venues";
+
+import "./AdminVenueView.scss";
+
+export interface SidebarOption {
+  id: string;
+  text: string;
+}
+
+enum SidebarOptions {
+  dashboard = "dashboard",
+  basicInfo = "basic_info",
+  entranceExperience = "entrance_experience",
+  advancedMapSettings = "advanced_map_settings",
+  ticketingAndAccess = "ticketing_and_access",
+}
+
+const sidebarOptions: SidebarOption[] = [
+  {
+    id: SidebarOptions.basicInfo,
+    text: "Start",
+  },
+  {
+    id: SidebarOptions.entranceExperience,
+    text: "Entrance",
+  },
+  {
+    id: SidebarOptions.advancedMapSettings,
+    text: "Advanced",
+  },
+  {
+    id: SidebarOptions.dashboard,
+    text: "Dashboard",
+  },
+  // TODO: Reintroduce when field is decided what to include
+  // {
+  //   id: SidebarOptions.ticketingAndAccess,
+  //   text: "Ticketing and access",
+  // },
+];
+
+const DEFAULT_TAB = sidebarOptions.findIndex(
+  (option) => option.id === SidebarOptions.dashboard
+);
+
+interface AdminVenueViewProps {
+  venue: Venue_v2;
```

Comment:
`AnyVenue` is used in the legacy admin panel which is a union from a bunch of concrete interfaces like PartyMapVenue, JazzbarVenue and etc. and they are all extending our `BaseVenue` type.

`Venue_v2` was created by the person who introduced admin v2. And looks like it shares the same fields like `BaseVenue`

> How does Venue_v2 differ from our standard Venue types?

`Venue_v2` is extending other interfaces which are used for the separate tabs in admin v2 creation. `BaseVenue` is one huge interface with all of the fields.

The way I see it we have 2 options.
1. Remove `Venue_v2` and split `BaseVenue` into small interfaces and use them in the tabs when creating venue.
2. Rename `Venue_v2` and all of it's small interfaces to `BaseVenue` and remove the old `BaseVenue` implementation.

Reply:
We still want to keep the concept of `BaseVenue`, but it doesn't need to be 'monolithic', it can definitely be composed of 'smaller chunks' (and IMO, probably should be). So I think that means I like option 2 that you presented above.

We need to be careful not to 'lose' any important fields in unifying these, as there may already be 'divergence' between `BaseVenue` and `Venue_v2`. I think (unless we know for sure it's unused/legacy/etc), that after this 'unification' `BaseVenue` should contain all of the fields that it currently does (even if that is just through 'composition' rather than defined directly within it), and possibly some extras (depending on what is in `Venue_v2`)

Eventually we probably want to 'align' all of the `_v2` types with their 'generic/common version like this, but that's obviously OOS for this PR.

Reply:
@0xdevalias @denisdimitrov are we going to reuse any components/types from V2 in V3 implementation? If not, maybe it's wise not to touch that part and delete it altogether, when we have somewhat decent V3 version? I'm looking into `venues.ts` and it looks like a completely distinct world. It will take some substantial amount of time & effort to rework all of that.

![image](https://user-images.githubusercontent.com/63194656/116282746-cf9aec80-a793-11eb-9bf7-9023fd79396d.png)

Reply:
> are we going to reuse any components/types from V2 in V3 implementation

@mike-lvov I believe V3 is just V2+updates/fixes/new designs, so I would assume yes.

---

> it looks like a completely distinct world. It will take some substantial amount of time & effort to rework all of that.

@mike-lvov From memory, the main aspect is that it's just broken up into more 'smaller chunks' along the way, I don't think they are actually dramatically distinct. I haven't deeply diffed it against our current types to be sure about that though.